### PR TITLE
feat:adding imported_at on public tables

### PR DESCRIPTION
**Pull request by ramiroJCB**

### CODE REVIEW

File: migrations/20220907164245_imported_at_publics.up.sql [hunk 1]
```diff
@@ -46,4 +46,48 @@ ADD COLUMN _mergestat_synced_at timestamp with time zone  DEFAULT now() NOT NULL
...
+	r.value->>'Class' as class,
+	r.value->>'Type' as type,
+	v as vulnerability,
+	v->>'VulnerabilityID' as vulnerability_id,
+	v->>'PkgName' as vulnerability_pkg_name,
+	v->>'InstalledVersion' as vulnerability_installed_version,
+	v->>'Severity' as vulnerability_severity,
+	v->>'Title' as vulnerability_title,
+	v->>'Description' as vulnerability_description,
+    trivy_repo_scans._mergestat_synced_at
```
Comment (amenowanna) [inline migrations/20220907164245_imported_at_publics.up.sql:89]:
```suggestion
        trivy_repo_scans._mergestat_synced_at
```

**Review by amenowanna — DISMISSED** [review]
Added a suggestion to fix indentation. Other than that this looks great!

File: migrations/20220907164245_imported_at_publics.up.sql (hunk 1 above)
Comment (ramiroJCB) [reply migrations/20220907164245_imported_at_publics.up.sql:89]: now this is interesting , in my local i have it formatted  .

**Review by patrickdevivo — APPROVED** [review]
looks great!

**Pull request merged**